library/vault

```diff
@@ -3,12 +3,12 @@ Maintainers: Chris Hoffman <choffman@hashicorp.com> (@chrishoffman),
              Matthew Irish <matthew@hashicorp.com> (@meirish)
 GitRepo: https://github.com/hashicorp/docker-vault.git
 
-Tags: 1.6.0-rc
+Tags: 1.6.0, latest
 Architectures: amd64, arm64v8, arm32v6, i386
-GitCommit: 7f6248ccbebf5f158f3a7d3923d2fc5d8adbaf17
+GitCommit: ba4a0bcb8341b04e565b83784a1acee9f7592f9a
 Directory: 0.X
 
-Tags: 1.5.5, latest
+Tags: 1.5.5
```

Comment:
Are 1.5.x and 1.4.x still actively supported?

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
(Something to think about / discuss post-merge :smile:)